oneview_redfish_toolkit/tests/api/test_capabilities_object.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2018) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import json
+
+from oneview_redfish_toolkit.api.capabilities_object import CapabilitiesObject
+from oneview_redfish_toolkit.tests.base_test import BaseTest
+
+
+class TestCapabilitiesObject(BaseTest):
+    """Tests for CapabilitiesObject class"""
+
+    with open('oneview_redfish_toolkit/mockups/oneview/'
+              'ServerProfileTemplate.json') as f:
+        server_profile_template = json.load(f)
+
+    with open('oneview_redfish_toolkit/mockups/redfish/'
+              'CapabilitiesObject.json') as f:
+        capabilities_mockup = json.load(f)
+
+    def test_class_instantiation(self):
```

Comment:
It is just a comment, not a truth problem, I believe this test is unnecessary because is just testing if the class instance has the same type than was put in the test itself.

Reply:
For what I could see, we make this type of tests to check any instantiate error. But once the following test will already get any error for instantiation, I believe we can remove the first test.

Reply:
@lhsvobodaj, any thoughts about this?

Reply:
It seems to be unnecessary. I believe we should direct our effort to create tests that use different data  (JSON content) and then check whether the object has been created as expected.